frontend/src/pages/DashboardPage/ChartComponenet/ChartComponent.jsx

```diff
@@ -17,10 +17,10 @@ const ChartComponent = forwardRef(({ chartData, sliderValue }, ref) => {
       new Set(chartData.map((item) => item.feature1))
```

Comment:
Have you checked for no overlapping CSS properties as it can cause confusion later on?

Reply:
I tried to delete the ones that are useless in Dashboard.css but feel free to look into it more maybe I missed something